### DISCUSSION TITLE
参考文献的平齐格式/行内引用

### DISCUSSION THREAD

**Discussion by HippoPikapii** [stone-zeng/fduthesis#248] [Q&A] [answered]
[参考文献的引文位置 #163](https://github.com/stone-zeng/fduthesis/issues/163)
上面这个issue提到了一般参考文献标号是上标形式，但是有时候需要平齐格式/行内引用。

目前该问题已经关闭了。我查找到另外一个办法，这里记录一下。

即：在需要的地方用 `\parencite{xxx}` 代替 `\cite{xxx}` 可以直接实现平齐格式引用，不需要其它修改。

**Comment by stone-zeng** [comment, marked as answer]
<img width="869" alt="Screen Shot 2022-03-23 at 20 13 32" src="https://user-images.githubusercontent.com/18233088/159696749-809d3829-c661-45ea-8e60-ac21aa68277a.png">

**Reply by sikouhjw** [reply]
不应该是“同左”吗……

**Reply by stone-zeng** [reply]
是我又傻了……
